Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/TwoQubitRB.py

```diff
@@ -73,7 +80,20 @@ def __init__(
         for i, qe in config["elements"].items():
             if "operations" not in qe:
                 qe["operations"] = {}
-        self._rb_baker = RBBaker(config, single_qubit_gate_generator, two_qubit_gate_generators, interleaving_gate)
+
```

Comment:
The docstring of the class breaks at the `returns` in measure_func

Reply:
I'm not sure how you found that it was broken, but I've tried to fix it. @TheoLaudatQM please let me know if it still looks broken

Reply:
I just checked it on Pycharm, it looks good now!